ee/candi/cloud-providers/vsphere/docs/ENVIRONMENT.md

```diff
@@ -6,122 +6,118 @@ title: "Cloud provider — VMware vSphere: Preparing environment"
 
 ## List of required vSphere resources
 
-* **User** with required set of [permissions](#permissions).
-* **Network** with DHCP server and access to the Internet.
+* **User** with required set of [permissions](#creating-and-assigning-a-role).
+* **Network** with DHCP server and access to the Internet
 * **Datacenter** with a tag in [`k8s-region`](#creating-tags-and-tag-categories) category.
-* **ComputeCluster** with a tag in [`k8s-zone`](#creating-tags-and-tag-categories).
-* **Datastore** with required [tags](#datastore-tags).
-* **Template** — [prepared](#building-a-vm-image) VM image.
+* **Cluster** with a tag in [`k8s-zone`](#creating-tags-and-tag-categories) category.
+* **Datastore** with required [tags](#datastore-configuration).
+* **Template** — [prepared](#preparing-a-virtual-machine-image) VM image.
 
-## Configuring vSphere
+## vSphere configuration
 
-The vSphere CLI called [govc](https://github.com/vmware/govmomi/tree/master/govc#installation) is designed to configure vSphere.
+### Installing govc
 
-### Setting up govc
+You'll need the vSphere CLI — [govc](https://github.com/vmware/govmomi/tree/master/govc#installation) to proceed with the rest of the guide.
 
-To configure the utility, set the following environment variables:
+After installation, set environment variables to work with vCenter:
 
 ```shell
 export GOVC_URL=example.com
-export GOVC_USERNAME=<USER_NAME>
-export GOVC_PASSWORD=<USER_PASSWORD>
+export GOVC_USERNAME=<username>@vsphere.local
+export GOVC_PASSWORD=<password>
 export GOVC_INSECURE=1
 ```
 
 ### Creating tags and tag categories
 
-VMware vSphere doesn't have *regions* and *zones*. It has Datacenters and ComputeClusters.
+VMware vSphere doesn't have "regions" and "zones". It has `Datacenter` and `Cluster` objects. To establish relation between these and "regions"/"zones" we'll use tags that fall into two tag categories. One for "region" tags and another for "zones tags".
```

Comment:
```suggestion
Instead of "regions" and "zones", VMware vSphere provides `Datacenter` and `Cluster` objects. We will use tags to match them with "regions"/"zones". These tags fall into two categories: one for "regions" tags and the other for "zones" tags.
```